controllers/keyword.go

```diff
@@ -0,0 +1,51 @@
+package controllers
+
+import (
+	"net/http"
+
+	. "google-scraper/helpers"
+	. "google-scraper/models"
+	"google-scraper/presenters"
+
+	"github.com/beego/beego/v2/core/logs"
+	"github.com/beego/beego/v2/server/web"
+)
+
+type KeywordController struct {
+	baseController
+}
+
+func (c *KeywordController) Show() {
+	web.ReadFromRequest(&c.Controller)
+	keyword, err := c.findKeyword()
+	if err != nil {
+		flash := web.NewFlash()
+		flash.Error(err.Error())
+		flash.Store(&c.Controller)
+		c.Ctx.Redirect(http.StatusFound, "/")
+		return
+	}
+
+	keywordPresenter, err := presenters.InitializeKeywordPresenter(keyword)
+	if err != nil {
+		logs.Error("Initializing presenter failed: ", err)
+	}
+
+	c.setAttributes(keywordPresenter)
+}
+
+func (c *KeywordController) findKeyword() (*Keyword, error) {
+	keywordId := c.Ctx.Input.Param(":id")
+	Id, err := StringToInt(keywordId)
+	if err != nil {
+		logs.Error("Converting String to Int failed: ", err)
+		return nil, err
```

Comment:
How about return user friendly message here? I'm not sure if the error message of `StringToInt()` (which is `strconv.ParseInt(str, 10, 64)` is user friendly.